config/application.rb

```diff
@@ -246,12 +246,27 @@ def dynflow
         if defined?(ForemanTasks)
           ForemanTasks.dynflow
         else
-          Dynflow::Rails.new(Foreman::Dynflow::Configuration.new)
+          ::Dynflow::Rails.new(nil, Foreman::Dynflow::Configuration.new)
```

Comment:
`Dynflow::Rails` was resolving to `Foreman::Dynlfow::Rails` which was not existing: the after_initialize block actually discovered this issue as it seems this code-path was not run in tests before.

Reply:
Thanks, I had to do something similar in https://github.com/theforeman/foreman/pull/4717/, can confirm it fixes the problem

Reply:
Yes, this should fix the issue you were facing there + making it work with later dynflow version as well that came with https://github.com/Dynflow/dynflow/commit/ff129a6b92eb209a82e51d177034b6f9187bff19#diff-56fbff8c6eb846b55588d78db914924aR10